### PR TITLE
Add `issue_template.md`

### PR DESCRIPTION
**Pull request by maclover7**
This appears to be a new feature of GitHub.

See these links for more details:
- https://github.com/dear-github/dear-github/issues/125
- https://github.com/owncloud/core/issues/new

[ci skip]

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+to keep the conversation linked together.
+
+### Other Information
+
+If there's anything else that's important and relevant to your pull
+request, mention that information here. This could include
+benchmarks, or other information.
+
+Finally, if your pull request includes code changes that only affect
+**documentation**, please amend your commit(s) to include `[ci skip]`
```
Comment (claudiob) [inline pull_request_template.md]: Would it be "documentation or comments" ?

Comment (maclover7) [inline pull_request_template.md]: :+1:

**Comment by pedelman** [comment]
Might be good idea to move these into the `.github` directory.

Also: https://github.com/blog/2111-issue-and-pull-request-templates

> If you're worried about the added clutter in the root directory of your project, we also added support for a .github/ folder. You can put CONTRIBUTING.md, ISSUE_TEMPLATE.md, and PULL_REQUEST_TEMPLATE.md files in .github/ and everything will work as expected.

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+### Steps to reproduce
+
+(Provide either a sample application or create an executable test script using one of the templates [here](https://github.com/rails/rails/tree/master/guides/bug_report_templates))
```
Comment (arthurnn) [inline .github/issue_template.md]: how about if we point to the actual guides page. http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#creating-a-bug-report ?

Comment (vipulnsward) [inline .github/issue_template.md]: :+1: on adding contributing link

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+these bugs have open GitHub issues, be sure to tag them here as well,
+to keep the conversation linked together.
+
+### Other Information
+
+If there's anything else that's important and relevant to your pull
+request, mention that information here. This could include
+benchmarks, or other information.
+
+Finally, if your pull request includes code changes that only affect
```
Comment (vipulnsward) [inline .github/pull_request_template.md]: This part can include the contributing to documentation link.

**Comment by maclover7** [comment]
Updated per comments.

**Comment by maclover7** [comment]
:tada:

**Pull request merged**